riscv/csrs.h

```diff
@@ -256,6 +256,15 @@ class mstatus_csr_t final: public base_status_csr_t {
 
 typedef std::shared_ptr<mstatus_csr_t> mstatus_csr_t_p;
 
+class mnstatus_csr_t final: public basic_csr_t {
+ public:
+  mnstatus_csr_t(processor_t* const proc, const reg_t addr);
+ protected:
+  virtual bool unlogged_write(const reg_t val) noexcept override;
+ private:
+  reg_t val;
```

Comment:
Hides the parent class's `val`. Instead call `basic_csr_t::unlogged_write()` from within `mnstatus_csr_t::unlogged_write()`.

Reply:
Oops, I should've deleted this when I changed the choice of base class.